Lombiq.HelpfulLibraries/Libraries/Validation/EmailValidationHelpers.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System;
+using System.Linq;
+using System.Net.Mail;
+
+namespace Lombiq.HelpfulLibraries.Libraries.Validation
+{
+    public static class EmailValidationHelpers
+    {
+        public static bool IsValidEmailAddress(string email)
+        {
+            try
+            {
+#pragma warning disable S1481
+                var mailAddress = new MailAddress(email);
+#pragma warning restore S1481

```

Comment:
```suggestion
                _ = new MailAddress(email);
```
You can use [discards](https://docs.microsoft.com/en-us/dotnet/csharp/fundamentals/functional/discards).